documentation-site/static/examples/avatar/sizes1.js

```diff
@@ -0,0 +1,26 @@
+import {Avatar} from 'baseui/avatar';
+import Code from '../../../pages/components/code';
+
+const scope = {Avatar};
+
+const code = `import * as React from 'react';
```

Comment:
Ideally, we want to 100% reuse the existing examples from `documentation-site/static/examples/` and keeping them as .js files. You can load them through webpack like this:

```js
const code = require('!!raw-loader!./example.js');
```